### DISCUSSION TITLE
Showcase CRUD example : filtering by column is not possible

### DISCUSSION THREAD

**Discussion by ritonglue** [primefaces/community#3173] [PrimeFaces] [answered]
Hello,
If you go to the crud example https://www.primefaces.org/showcase/ui/data/datatable/crud.xhtml
you can see that column filtering is enabled per column with the filterBy attribute. For example:
`<p:column headerText="Name" sortBy="#{product.name}" filterBy="#{product.name}">`
Unfortunately, the style crud-demo disables this feature.

It took me a while to find out why filtering by column was disabled this way. Is it on purpose ?

**Comment by melloware** [comment, marked as answer]
It looks like PrimeTek did thta on purpose just to show off "global filtering" but I will enable it.
